taskograph.Helpers/TestHelpers.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace taskograph.Helpers
+{
+    public static class TestHelpers
+    {
+        public const string TEST_PARAMETER = "test_parameter";
```

Comment:
Uwaga na przerost formy na treścią - jeśli masz jeden const, to może nie warto przenosić go do osobnego pliku